packages/auth-construct/API.md

```diff
@@ -22,7 +20,6 @@ export type AmazonProviderProps = Omit<aws_cognito.UserPoolIdentityProviderAmazo
 // @public
 export class AmplifyAuth extends Construct implements ResourceProvider<AuthResources> {
     constructor(scope: Construct, id: string, props?: AuthProps);
-    addTrigger: (event: TriggerEvent, handler: IFunction | AmplifyFunction) => void;

```

Comment:
Removing this because it exposes some of our higher-level abstractions at the CDK layer (ie AmplifyFunction). Instead, customers can use `authConstruct.resources.userPool.addTrigger()`. Alternatively, I could keep this method but have the input only allow IFunction and not AmplifyFunction.